### DISCUSSION TITLE
Automatically change the "default communications device" when changing the playback device.

### DISCUSSION THREAD

**Discussion by Wim-De-Clercq** [File-New-Project/EarTrumpet#763] [Ideas]
When right clicking the ear trumpet system tray icon you can change the output device
![image](https://user-images.githubusercontent.com/4599667/108818309-778c1200-75b9-11eb-85c2-6a215a4d044e.png)

It would be nice if that also changes both the defaults, or maybe have a setting to allow it to do so.
![image](https://user-images.githubusercontent.com/4599667/108818385-925e8680-75b9-11eb-9147-0a1feb0b8b79.png)

**Comment by CosimoMatassini** [comment]
https://audioswit.ch/er

**Comment by andrewjmetzger** [comment]
There is also https://soundswitch.aaflalo.me/ , which I personally use alongside EarTrumpet. Although I've never used Audio Switcher, I think SoundSwitch accomplishes something very similar using context menus and (optionally) hotkeys. You can even prevent undesired audio devices from being used by deselecting them in the **Playback, Recording**, and/or **Profiles** tabs. 

Here's a screenshot of the relevant settings:
![image](https://user-images.githubusercontent.com/590246/122616617-b65a4200-d058-11eb-96ad-c88ebdab800d.png)
